src/Test.UnitTests.Sarif/Visitors/SuppressVisitorTests.cs

```diff
@@ -125,12 +125,12 @@ public void SuppressVisitor_ShouldFlowPropertiesCorrectly()
 
                 if (timestamps && suppression.TryGetProperty("timeUtc", out DateTime timeUtc))
                 {
-                    timeUtc.Should().BeCloseTo(DateTime.UtcNow);
+                    timeUtc.Should().BeCloseTo(DateTime.UtcNow, 500);
```

Comment:
>500 [](http://example.com/codeflow?start=64&length=3)

Capture this in a constant somewhere. Do me a favor and grepo the code for BeCloseTo to see if we've got other problems of this kind lurking.

Reply:
fixed and searched the "BeCloseTo", none left.